.travis.yml

```diff
@@ -41,12 +42,19 @@ before_install:
   - wget https://github.com/neo4j-drivers/seabolt/releases/download/v1.7.4/seabolt-1.7.4-Linux-ubuntu-$(lsb_release -rs).deb
   - sudo dpkg -i seabolt-1.7.4-Linux-ubuntu-$(lsb_release -rs).deb
   - rm seabolt-1.7.4-Linux-ubuntu-$(lsb_release -rs).deb
+  # Install oracle dependencies
+  - sudo apt-get install unzip libaio1
+  - wget https://maxnilz.com/instantclient-basiclite-linux.x64-18.5.0.0.0dbru.zip
```

Comment:
We should not be circumventing licenses. If there's no way for the project to accept a license, then we cannot include the software in the project.

Also, any files should be hosted by a trusted and dependable service.

Reply:
resolved